EXCLUSION.md

```diff
@@ -1,3 +1,44 @@
+
+Apart from the reasons stated below, these exclusions are also intended to prevent TwoFactorAuth.org from being added to any professional or academic web filters, where automatic addition of sites supersedes human checks during which the goal of our project would be weighted higher than the link to a potentially questionable site.
+
+If you want to make a copy (fork) of our site to list any of these sites you're welcome to do so as long as you comply with our [license](LICENSE).
+
+## Categories
+
+*   #### Pornographic sites
+
+    As the volunteers of [2factorauth](https://github.com/2factorauth) need to go through all sites to validate that the site/service provides 2FA to it's users and verify that the pull request meets our style guidelines we've chosen not to list sites that primarily host and serve pornographic content.
+    We've also chosen to do this so that we don't have to go through potentially disturbing or illegal material.
+
+*   #### Self-hosted services
+
+    The category "self-hosted services/sites" includes all types of software that's designed to be hosted by the end user. These are excluded from listing unless all these points are met:
+
+    1. All [Site Criteria](CONTRIBUTING.md#site-criteria) and [Guidelines](CONTRIBUTING.md#guidelines) are met
+
+    2. Public interest and public accessibility are given  
+
+    3. The site has an independent authentication database
+
+    4. The underlying service supports 2FA natively or through a first-party (developer) plugin
+
+    There are several resons for why we've opted to not list such sites and services.
+
+    -   TwoFactorAuth.org is targeted towards consumers and not website administrators. As of right now, very few general internet users choose to host their own websites and services. Therefore we think that the effort on our part to maintain such a list of self hosted alternatives would outweigh the minimal theoretical change in the internet landscape by listing such services.
+
+    -   If the core project doesn't support two factor authentication but instead through plugins it would mean that we would have to list all plugins that enable two factor authentication for the service. Something that is currently impossible with our website layout.
+
+    If you have any questions regarding whether or not a site qualifies for listing, simply open an issue and we'll take a look.
+
+*   #### Potential controversial sites 
+
+    Any site that could damage the reputation of TwoFactorAuth.org and/or lead to any controversy with either the maintainers or users, should not be listed.
+    The group of active maintainers will decide on whether to include or exclude a site within a specific timeframe. Once an exclusion was decided upon, the site will be listed in the section below and - if a pull request exists - it will be closed.
+    This decision will also be in effect for pull requests that already existed before this exclusion documentation was published. 
```

Comment:
I don't think a "retroactivity" clause is really necessary for this document, especially if it is only going to apply to a single subsection.
```suggestion
```